src/c/core/session/write_access.c

```diff
@@ -64,8 +66,10 @@ uint16_t uxr_buffer_reply(
         WRITE_DATA_Payload_Data payload;
         rv = uxr_init_base_object_request(&session->info, replier_id, &payload.base);
         uxr_serialize_WRITE_DATA_Payload_Data(&ub, &payload);
+        UXR_PREPARE_INTERPROCESS(session, replier_id, &ub, (uint16_t) payload_size);
```

Comment:
Should we use `(uint16_t) len` instead of `(uint16_t) payload_size`?

Reply:
That's true